### ISSUE TITLE
how to specify JWT token or custom header?

### ISSUE THREAD

**Issue by nsisodiya** [lucasconstantino/graphiql-online#6]
This tool is not much useful unless we specify JWT token. Kindly let us know if we can specify the token.

**Comment by EverWinter23** [comment]
@nsisodiya I agree, would be great if we could do that. Otherwise it's an excellent tool, just need to add JWT support.

**Comment by AL-HAYALI** [comment]
We can't specify API token, Especially with Shopify without it it's useless.
